### ISSUE TITLE
[Bug]: Validation of inner component resets cursor when outer component has LoadingIndicator

### ISSUE THREAD

**Issue by cpiber** [Megabit/Blazorise#5727]
### Blazorise Version

1.6.1

### What Blazorise provider are you running on?

None

### Link to minimal reproduction or a simple code snippet

https://github.com/cpiber/BlazoriseReproductions/tree/memoedit-validation-cursor, relevant files BaseProject/BlazoriseReproduction/Client/Pages/{Index.razor,Component.razor}

Same happens on [previous commit](https://github.com/cpiber/BlazoriseReproductions/tree/bbb0894ec6acb9009ab3ca35edd9d44c1a822d04) when two `Fields` are nested

### Steps to reproduce

Enter 4 characters into the text edit. Move the cursor into the middle. Type another character to trigger the validation.

### What is expected?

The cursor should remain where it is.

### What is actually happening?

The cursor will jump to the end.

### What browsers do you see the problem on?

Chrome, Firefox

### Any additional comments?

I assume it has to do with the rerendering of the outer component, it only happens when the Text is bound to a value on the other component. I could only get it to happen when the TextEdit (or MemoEdit) is inside a Field, and the outer component must wrap the inner one in a LoadingIndicator or Fields.

**Comment by stsrki** [comment]
This is how Blazor works, so there is not much around it. When value os two-way binded it will refresh the rendering tree and the cursor will jump to the end since Blazor does not keep track of it.

We could try to handle it with the help of JS, but it would still not work 100%. Wi tried that approach in the past. My advice is to make the component update value only on losing the focus.

Use this code

```
<TextEdit Text="@Val" TextChanged="@ValChanged" Immediate="false" />
```

**Comment by cpiber** [comment]
Hm, thank you for your insight, we'll try that, that's too bad, but I believe I experienced a similar thing with react, so that would make sense

**Comment by stsrki** [comment]
Yes, all front-end frameworks can have similar problems, including React. But unlike Blazor, React have a direct way to JS, so it is a lot easier for them to make it less obvious and easier to fix.
